src/cluster/dcommands/admin/ccommand.ts

```diff
@@ -752,10 +756,13 @@ export class CustomCommandCommand extends GuildCommand {
             return undefined;
 
         const command = await context.database.guilds.getCommand(context.channel.guild.id, commandName);
-        if (command === undefined)
-            return { name: commandName };
+        if (command !== undefined)
+            return { name: command.name, command };
 
-        return { name: command.name, command };
+        if (commandName.length > 100)
+            return this.error('Command names cannot be longer than 100 characters');
```

Comment:
I feel like this is not necessary in this part of the code. Might be useful higher in the logic though

Reply:
I think it makes sense here, it fits with the pattern of validation for the other bits, and going forwards this 100 character limit should be applied to any new ccommands that get made, for which every route goes through here.
I would have put it at the start of this method, but that would then prevent you interacting with any commands that already exist over 100 characters, so doing the check only when the command doesnt already exist is the best place for it imo.